dask/array/tests/test_fft.py

```diff
@@ -137,6 +137,7 @@ def test_wrap_ffts(modname, funcname, dtype):
 
     wfunc = fft_wrap(func)
     assert wfunc(darrc).dtype == func(nparrc).dtype
+    assert wfunc(darrc).shape == func(nparrc).shape
```

Comment:
OK, so I guess this gets back to your original question about if we should support scipy `rfft` functions, and, if not, how to exclude them.  It seems like the current chunk_fn for rfft is specific to `numpy`.  Do we always ask the user to provide a chunk function of their own?